### PR TITLE
Ensure described topic exists

### LINKED ISSUE #49: describe topic doesn't check if topic exists
(esque) Garretts-MacBook-Pro:esque garrettthomas$ esque describe topic fsdjklhaslhjke
Topic: fsdjklhaslhjke
Config:
  cleanup.policy:                          delete
  compression.type:                        producer
  delete.retention.ms:                     1 day
  file.delete.delay.ms:                    1 minute
  flush.messages:                          9223372036854775807
  flush.ms:                                292471208 years
  follower.replication.throttled.replicas: ''
  index.interval.bytes:                    4.00 kib
  leader.replication.throttled.replicas:   ''
  max.message.bytes:                       976.57 kib
  message.downconversion.enable:           true
  message.format.version:                  2.2-IV1
  message.timestamp.difference.max.ms:     292471208 years
  message.timestamp.type:                  CreateTime
  min.cleanable.dirty.ratio:               0.5
  min.compaction.lag.ms:                   
  min.insync.replicas:                     1
  preallocate:                             false
  retention.bytes:                         None
  retention.ms:                            1 week
  segment.bytes:                           1.00 Gib
  segment.index.bytes:                     10.00 Mib
  segment.jitter.ms:                       
  segment.ms:                              1 week
  unclean.leader.election.enable:          false

### PR DESCRIPTION
**Pull request by hfjn**
Fixes #49. Still needs testing. Will add that test after #54 is merged.

### CODE REVIEW

**Review by garrettthomaskth — APPROVED** [review]
looks great, no suggestions!

**Review by Bibob7 — APPROVED** [review]
Looks good and works! :)

File: .pytest_cache/v/cache/nodeids [hunk 1]
```diff
@@ -0,0 +1,42 @@
+[
```
Comment (swenzel) [inline .pytest_cache/v/cache/nodeids]: I think we should add `.pytest_cache` to `.gitignore`?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/hfjn/.local/share/virtualenvs/esque-YsmoIp6f/bin/python"
```
Comment (swenzel) [inline .vscode/settings.json]: This is probably not supposed to end up here either ;)

**Review by swenzel — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (hfjn) [reply .vscode/settings.json]: God damnit. How did that came back?

**Review by swenzel — APPROVED** [review]

**Pull request merged**